packages/admin/src/pages/build-application/[applicationId]/[sectionId]/index.page.tsx

```diff
@@ -45,7 +86,29 @@ const EditSectionPage = ({
   section,
   grantApplicationName,
   applicationId,
+  scrollPosition,
+  resolvedUrl,
+  csrfToken,
 }: InferProps<typeof getServerSideProps>) => {
+  const formRef = useRef<HTMLFormElement>(null);
+  const [newScrollPosition, setNewScrollPosition] = useState(
+    scrollPosition ?? 0
+  );
+  const formAction =
+    resolvedUrl.split('?')[0] + `?scrollPosition=${newScrollPosition}`;
+
+  function handleOnUpDownButtonClick() {
+    setNewScrollPosition(window.scrollY);
+    formRef?.current?.submit();
+  }
+
+  useEffect(() => {
```

Comment:
I think scrolling on load is an actual use case for `useLayoutEffect` - it'll ensure the scroll completes synchronously before the page is rendered, which will prevent any weird flickering behaviour where the page loads for a second and _then_ scrolls.

Reply:
Ohhhh neat will test this out